lib/active_admin/views/components/tabs.rb

```diff
@@ -15,17 +15,26 @@ def build(&block)
       end
 
       def build_menu_item(title, options, &block)
-        fragment = options.fetch(:id, title.parameterize)
+        fragment = options.fetch(:id, fragmentize(title))
+
         html_options = options.fetch(:html_options, {})
         li html_options do
           link_to title, "##{fragment}"
         end
       end
 
       def build_content_item(title, options, &block)
-        options = options.reverse_merge(id: title.parameterize)
+        options = options.reverse_merge(id: fragmentize(title))
```

Comment:
`URI.encode` is a good catch, but I think with it we no longer need `fragmentize` and can do:
```suggestion
        options = options.reverse_merge(id: URI.encode(title))
```

Reply:
But it will brake existing fragments.
```ruby
'a b'.parameterize #=> "a-b"
URI.encode('a b') #=> "a%20b"
```

Reply:
Some apps can still rely on parameterized titles.